### ISSUE TITLE
Feature request: Implement 2 features from Neo Store (Privavy Visualizations & Vulnerability notifications)

### ISSUE THREAD

**Issue by huuhaa** [Iamlooker/Droid-ify#318]
**Is your feature request related to a problem? Please describe.**

Not a problem, but having these 2 features would improve usage and offer extra information

**Describe the solution you'd like**

I would like that [Privacy Visualizations](https://refubium.fu-berlin.de/handle/fub188/38148) from [Neo Store](https://github.com/NeoApplications/Neo-Store) would be implemented to Droid-ify.

Another nice feature would be to implement feature to give user notification when app has known vulnerability, but first it's good to wait until[ issue related this functionality has been fixed](https://github.com/NeoApplications/Neo-Store/issues/456). (Link does include video about functionality)

**Describe alternatives you've considered**
Privacy Visualization isn't "must have", but nice to have feature. Vulnerability warnings on the other hand, while could be at anti features, notification will be noticed a lot more effectively.
**Additional context**
Pics how privacy visualizations are shown:

![PV1](https://user-images.githubusercontent.com/469355/230401315-a8811a80-fbc0-45f4-98b5-a6f5e1ba7d23.jpg)

![PV2](https://user-images.githubusercontent.com/469355/230401347-6a6f1dc4-b1f7-46e8-817f-7dba950092a4.jpg)

**Comment by Justauser0** [comment]
Any update

**Comment by Iamlooker** [comment]
> Any update

This will take some time, I am currently very busy. And will finish work on the current PR first

**Comment by candroid-man** [comment]
Yep, it's the only thing holding me back from Droid-ify, I much prefer the simpler UI that it has to offer but it needs more features like this
